### PR TITLE
Improve default screen sharing FPS, limiting capturing surface

### PR DESCRIPTION
**Pull request by davidzhao**
Previously when we didn't constrain screen share resolution, screen share tracks weren't resized to a reasonable size prior to publishing. Having to encode a huge frame caused FPS to be very low (often just 1-5 fps).

Because that's rather confusing behavior, we are now defaulting to 1080p when capturing

### CODE REVIEW

**Comment by lukasIO** [comment]
When removing the default resolution other than fixing the Safari bug we also did so in order to enable users to capture at native screen sizes. When applying a default like this, that won't be possible anymore.

I think both makes sense: having a reasonable default, but also allowing users to capture uncapped resolution if they wish to do so. Maybe a `ScreenSharePreset.Uncapped` or something similar would allow for that? Not sure

**Comment by davidzhao** [comment]
@lukasIO I've added `original`. lmk what you think.

**Comment by lukasIO** [comment]
nice! was thinking whether explicitly passing `null` would be an option, too, but I somehow prefer the original preset!

**Review by lukasIO — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "build:watch": "rollup --watch --config rollup.config.js",
     "build-docs": "typedoc",
     "proto": "protoc --es_out src/proto --es_opt target=ts -I./protocol ./protocol/livekit_rtc.proto ./protocol/livekit_models.proto",
-    "sample": "vite example -c vite.config.js",
+    "sample": "vite example -c vite.config.mjs",
```
Comment (lukasIO) [inline package.json:43]: thanks 🙏

**Review by lukasIO — APPROVED** [review]

**Pull request merged**